dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/specific/uploadconfig/elements/UploadConfigElement.kt

```diff
@@ -0,0 +1,25 @@
+package org.dataland.frameworktoolbox.specific.uploadconfig.elements
+
+/**
+ * A single element of a MLDT View-Configuration (either Cell or Section)
+ */
+sealed interface UploadConfigElement {
```

Comment:
The sealed attribute is set so that it is required that at compile time the implementations of this interface are know, correct?

Reply:
I am not sure. @MarcTM01  can you explain to us why you used a sealed interface here?

Reply:
Perfect, thank you!